stack.yaml

```diff
@@ -7,5 +7,11 @@ packages:
   - .
   - template
   - ligo/haskell
+  - ../morley/code/morley
+  - ../morley/code/lorentz
+  - ../morley/code/cleveland
+  - ../morley/code/morley-client
+  - ../indigo
+  - ../morley-ledgers/code/morley-ledgers
```

Comment:
This MR cannot be merged until latest version of `morley`, `indigo`, `morley-ledgers`, and MR #144 are merged.

Reply:
To be specific, these are the MRs:

- lorentz: https://gitlab.com/morley-framework/morley/-/merge_requests/759
- indigo: https://gitlab.com/morley-framework/indigo/-/merge_requests/15
- morley-ledgers: https://gitlab.com/morley-framework/morley-ledgers/-/merge_requests/33